src/inspector_agent.cc

```diff
@@ -110,9 +110,11 @@ void SendTargentsListResponse(inspector_socket_t* socket, int port) {
   char buffer[sizeof(LIST_RESPONSE_TEMPLATE) + 4096];
   char title[2048];  // uv_get_process_title trims the title if too long
   int err = uv_get_process_title(title, sizeof(title));
-  ASSERT_EQ(0, err);
+  if (err != 0) {
+    strcpy(title, "Node.js");
```

Comment:
Can you use `snprintf()`?  Most static analysis tools will complain about `strcpy()` (even if it's harmless here.)